docs/installguide/release_notes.rst

```diff
@@ -19,6 +19,28 @@ Bug fixes
  * Use current year in parts of footer :url-issue:`5112`
  * Handle socket.error: Fix some cases where KA Lite fails to start due to a previous unclean shutdown :url-issue:`5132`
 
+Known issues
+^^^^^^^^^^^^
+
+Please note that issues with **content packs** are not related to the software
+itself but is being fixed and updated along side our release.
+
+Watch
+`dev@learningequality.org <https://groups.google.com/a/learningequality.org/forum/#!forum/dev>`__
+for announcements.
+
+ * **Content packs** 1800 questions (assessment items) inside exercises (English version) have been removed due to an upstream data bug. The count is lower in other languages. There are 29,839 assessment items, so it's 6% of questions (not exercises) that are missing. In terms of exercises, they are all there.
```

Comment:
`1800 questions (assessment items) inside exercises (English version) have been removed due to an upstream data bug`

= >

`1800 outdated questions (assessment items) inside exercises were removed. (...). All exercises are still there.